rdtools/test/conftest.py

```diff
@@ -9,30 +9,32 @@
 
 import rdtools
 
-rdtools_base_version = \
-    parse_version(parse_version(rdtools.__version__).base_version)
+# rdtools_base_version = \
+#     parse_version(parse_version(rdtools.__version__).base_version)
 
 
 # decorator takes one argument: the base version for which it should fail
 # for example @fail_on_rdtools_version('3.0.0') will cause a test to fail
 # on rdtools versions 3.0.0, 3.0.0-alpha, 3.1.0, etc
-def fail_on_rdtools_version(version):
```

Comment:
@mdeceglie - pkg_resources is depricated for python 3.12. I didn't find a single instance in the tests where this decorator is used. Should we deprecate the decorator or refactor it to work with the importlib.metadata package?